### ISSUE TITLE
Consolidate Python and Deb version

### ISSUE THREAD

**Issue by vpetersson** [WoTTsecurity/agent#96]
As it stands right now, we have two different versions of the agent (in the same repo):

* The Python version
* The Debian package version

let's consolidate the two are coupled right now anyway.

**Comment by a-martynovich** [comment]
took this

**Comment by a-martynovich** [comment]
@vpetersson How do we manage versions?

In #51 you suggested to use short hash as part of version number. That means we'll have e.g. `0.1.5.aabbccdd` instead of `0.1.5.9`.

But the first three parts of the version (`0.1.5`) should still be managed manually. I changed `0.1.4` to `0.1.5` when I made #35 because it was sort of a major change.

I'm suggesting to put this manually managed version into a `VERSION` file at the root of the repo. This file will be used to build complete version string both for setuptools (setup.py) and debian package (changelog).
